src/citrine/informatics/executions/design_execution.py

```diff
@@ -64,6 +64,8 @@ class DesignExecution(Resource['DesignExecution'], Pageable, AsynchronousObject)
 
     score = properties.Object(Score, 'score')
     """:Score: score by which this execution was evaluated"""
+    max_candidates = properties.Optional(properties.Integer, 'max_candidates')
```

Comment:
I don't think we need to store this as part of the design execution unless users ask for it. Part of my hesitation is it would require us to persist it on the backend, which feels like overkill for an execution parameter which just limits the number of results.

Reply:
Agreed; I'll take it out